spec/models/alchemy/picture_spec.rb

```diff
@@ -227,6 +227,13 @@ module Alchemy
         end
       end
 
+      context "with 'not_used' as argument" do
+        it 'should call the .not_used scope' do
+          expect(Picture).to receive(:not_used).and_return(picture)
+          expect(Picture.filtered_by('not_used')).to eq(picture)
+        end
+      end
```

Comment:
This test is not testing that if a picture is assigned it does not appear in the list.